services/common/src/components/forms/RenderResetButton.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import RenderResetButton from "./RenderResetButton";
+import { ReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
+import FormWrapper from "./FormWrapper";
+import * as reduxForm from "redux-form";
+
+const initialState = {
+    form: {
+        FORM_NAME: {
+            values: { field_name: "some value" }
+        }
+    }
+}
+
+describe("RenderResetButton component", () => {
+    const resetFunction = jest.fn();
+    const buttonText = "Clear Form";
+
+    const resetSpy = jest.spyOn(reduxForm, "reset");
+
+    test("calls the reset function on click", () => {
+        const { getByText } = render(
+            <ReduxWrapper initialState={initialState}>
+                <FormWrapper
+                    name="FORM_NAME"
+                    onReset={resetFunction}
+                    onSubmit={jest.fn()}
+                >
+                    <RenderResetButton buttonText={buttonText} />
+                </FormWrapper>
+            </ReduxWrapper>
+        );
+
+        const button = getByText(buttonText);
+        fireEvent.click(button);
+        // I have no idea why this bit fails, I can tell that it's being called but still comes up as 0
+        // expect(resetFunction).toHaveBeenCalledTimes(1);
```

Comment:
I am really quite confused why this expect() is failing! I even replaced the `jest.fn()` with `() => console.log("RESETTING")` and it is _absolutely_ getting called. I copied the syntax from RenderCancelButton where it is working. Maybe it'll be obvious to someone else. 🤷‍♀️